resources/assets/coffee/react/profile-page/medals.coffee

```diff
@@ -32,6 +33,15 @@ ProfilePage.Medals = React.createClass
 
     _.includes @_achieved, id
 
+  _getDateAchieved: (id) ->
```

Comment:
at this point it's easier to just pass the whole user's own achievement for the particular medal.